src/sharding/ShardingManager.js

```diff
@@ -139,6 +157,15 @@ class ShardingManager extends EventEmitter {
         return;
       }
 
+      if (this.endShard !== 'auto') {
+        Array(this.endShard + 1 - this.startShard).fill().map((x, i) => i + this.startShard)
+          .forEach(i => {
```

Comment:
you need to change this to a for..of, we don't use forEach in this library by convention.

Reply:
Or just a normal for loop, starting at `startShard` ending at `endShard`.